browser/scss/main.scss

```diff
@@ -83,8 +80,8 @@ md-card md-card-title {
   margin-top: 2px;
 }
 
-#plots .btn-flat {
-  margin-left: 45%;
+#plots.btn-flat {
+  margin-left: 45vw;
```

Comment:
I _think_ percentages are relative to the parent container while viewport is relative to the window size.